extensions/wikia/TrackingOptIn/js/trackingOptIn.js

```diff
@@ -44,9 +60,54 @@ define('wikia.trackingOptIn', [
 		window.Wikia.consentQueue.push(callback);
 	}
 
+	function geoRequiresTrackingConsent() {
+		return geoRequiresConsent;
+	}
+
+	function getPrebidGlobalVendorList() {
+		var request = new XMLHttpRequest();
+
+		log('Requesting IAB Global Vendor List', log.levels.debug, logGroup);
+
+		request.open('GET', prebidVendorListUrl, false);
+		request.send(null);
+
+		return JSON.parse(request.responseText);
+	}
+
+	function getPrebidConsentString(optIn) {
+		var cookie = cookies.get(prebidConsentStringCookie);
+
+		if (cookie) {
+			log('Serving consent string from cookie', log.levels.debug, logGroup);
+			return cookie;
+		}
+
+		var consentString,
+			consentData = new trackingOptInModal.ConsentString();
+
+		consentData.setGlobalVendorList(getPrebidGlobalVendorList());
+		consentData.setPurposesAllowed(optIn ? prebidPurposesList : []);
```

Comment:
Can we create this array based on `prebidVendorsList`?

Reply:
Done.